packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -828,6 +832,200 @@ describe("Integration tests", () => {
         );
       });
     });
+    describe("revokeCertifiedAttribute", async () => {
+      const attributeId: AttributeId = generateId();
+      const correlationId = generateId();
```

Comment:
As regards the `correlationId` I replaced it with the `generateId()`, while for the `attributeId` I cannot do it, because within some tests, I need it to always remain the same as the `attribute` under the `describe`

Reply:
> As regards the `correlationId` I replaced it with the `generateId()`, while for the `attributeId` I cannot do it, because within some tests, I need it to always remain the same as the `attribute` under the `describe`

You can always get the ID from the `attribute` object, whenever you use `attributeId` you can replace it with `attribute.id`